src/main/java/org/moditect/jfrunit/JfrAnalysisResults.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020 - 2021 The JfrUnit authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.moditect.jfrunit;
+
+import java.util.List;
+
+import org.openjdk.jmc.flightrecorder.rules.IResult;
+
+public class JfrAnalysisResults {
```

Comment:
Perhaps this could implement the `List<IResult>` (delegating to `results`). Groovy/Spock checks could be a bit easier then, I think.

Reply:
I did not think about the Groovy/Spock usage. Maybe I should add some tests for Groovy as well?

Reply:
If we just delegate to the underlying List, then there is probably little value in having this Class? I think it will be cleaner just to use a `List<IResult>` directly

Reply:
I can try to add a Spock test later once the things will settle bit more. We will see how it will look like. Re this class I think it might have value if you expect it will deliver data beyond the list of `IResult` itself. So I would say it is for your consideration.

Reply:
I have removed `JfrAnalysisResults` and am returning a `List<IResult>`